core-java/src/test/java/org/baeldung/java/sorting/SetSort.java

```diff
@@ -0,0 +1,54 @@
+package org.baeldung.java.sorting;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.TreeSet;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class SetSort {
+
+    private HashSet<Integer> integers;
+    private TreeSet<Employee> employees;
```

Comment:
Any particular reason why we are using both HashSet and TreeSet. Can we just use HashSet since we are trying to show how to *sort* *Set* ?